bridge/resources/resource.go

```diff
@@ -74,8 +74,43 @@ func (r *Resource) GetHref() string {
 	return r.Href
 }
 
+type SupportedOperation int
+
+const (
+	SupportedOperationRead SupportedOperation = 0x1 << iota
+	SupportedOperationWrite
+	SupportedOperationObserve
+)
+
+func (o SupportedOperation) HasOperation(operation SupportedOperation) bool {
+	return o&operation != 0
+}
+
+func (r *Resource) SupportsOperations() SupportedOperation {
+	var operations SupportedOperation
+	if r.getHandler != nil {
+		operations |= SupportedOperationRead
+	}
+	if r.postHandler != nil {
+		operations |= SupportedOperationWrite
+	}
+	if r.PolicyBitMask&schema.Observable != 0 {
+		operations |= SupportedOperationObserve
+	}
+	return operations
+}
+
+func (r *Resource) SetResourceTypes(resourceTypes []string) {
```

Comment:
Resource types are changeable in runtime?

Reply:
Yes. For WoT it can be changed.